include/deal.II/lac/precondition.h

```diff
@@ -976,7 +1018,7 @@ class PreconditionChebyshev : public Subscriptor
     /**
      * Constructor.
      */
-    AdditionalData(const unsigned int degree              = 0,
+    AdditionalData(const unsigned int degree              = 1,
```

Comment:
This is a follow-up change to #7708, isn't it?

Reply:
Yes, I had forgotten that change there and one of the exceptions I introduced here revealed this issue in a test.